x/nodes/keeper/params.go

```diff
@@ -90,10 +90,20 @@ func (k Keeper) RelaysToTokensMultiplier(ctx sdk.Ctx) sdk.Int {
 	return sdk.NewInt(1000) // todo parameterize
 }
 
-func (k Keeper) NodeCutOfReward(ctx sdk.Ctx) sdk.Int {
-	daoAllocation := k.DAOAllocation(ctx)
-	proposerAllocation := k.ProposerAllocation(ctx)
-	return sdk.NewInt(100 - daoAllocation - proposerAllocation)
+func (k Keeper) NodeCutOfReward(ctx sdk.Ctx, reward sdk.Int) (toNode sdk.Int, toFeeCollector sdk.Int) {
```

Comment:
NodeReward would be a better name?

Reply:
Also `nodeReward` and `feesCollected` or `collectedFees` or `fees` would make sense for the return values.

Reply:
Will fix!